### ISSUE TITLE
Saved Actions can't be saved to file

### ISSUE THREAD

**Issue by Discipol** [Kanatoko/MonjaDB#14]
I am looking for a way to load the saved actions from an external file. I have a GUI for my project that edits mongo and exports the commands to a txt folder. I would love to be able to load this in monjadb! If I can save Saved Actions to a file instead of wherever, I could edit it using my project's tool.

I would also like to reopen the whitespace issue and suggest you split the txt by ;\r|\n instead of just \r|\n

My first Github post :dancer:

**Comment by Kanatoko** [comment]
Hello, Thank you for your feedback.

Saved actions are composed as a tree structure.

Do you want to save(or load) each item(action) as one file?
Or, whole items(actions) as one file?

**Comment by Discipol** [comment]
Greetings! I would like to save/load each item(Action) from an external txt file at a specified path, perhaps with an option to save it in the hidden cache you use right now instead of using the external txt file.

Alternatively, save the entire action file in a xml file because that has a structure. JSON works as well.

I have an editor tool for my db and i want to save it into a txt so when I mess up my database, I can just run an action(with that file) that wipes the database and inserts the things my tool saved.

Thank you for your reply!

**Comment by Kanatoko** [comment]
Thank you for reply.

How do you think about copy/pasting via clipboard?
-Open your txt file with some editor
-Select all then copy
-Close editor
-Create new action item on MonjaDB
-Input name
-Paste data as 'Actions'

**Comment by Discipol** [comment]
Yeah that is what I have to do right now. When I paste the test from an utf8, there are no \r or \n. After I save and ten reload the action it formats it properly, but the steps themselves are a bit frustrating because I have to do it EACH TIME.

One more problem related to this. I can't replicate the problem, but if a collection is selected, if the _id IS NOT the first column, then selecting it DOES NOT load it in the Document Editor or JSON editor.
